pep-0648.rst

```diff
@@ -1,8 +1,7 @@
 PEP: 648
 Title: Extensible customizations of the interpreter at startup
 Author: Mario Corchero <mariocj89@gmail.com>
-Sponsor: Pablo Galindo
-BDFL-Delegate: XXXX
+Sponsor: Pablo Galindo <pablogsal@python.org>
```

Comment:
I checked with Pablo and he said let's add it here. 👍

Reply:
Thanks ❤️